pkg/gds/gds_test.go

```diff
@@ -4,9 +4,107 @@ import (
 	"context"
 	"time"
 
+	"github.com/trisacrypto/directory/pkg/gds/models/v1"
 	api "github.com/trisacrypto/trisa/pkg/trisa/gds/api/v1beta1"
+	pb "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
 )
 
+// TestRegister tests that the Register RPC correcty registers a new VASP with GDS.
+func (s *gdsTestSuite) TestRegister() {
+	s.LoadEmptyFixtures()
+	defer s.ResetEmptyFixtures()
+	require := s.Require()
+	ctx := context.Background()
+	refVASP := s.fixtures[vasps]["d9da630e-41aa-11ec-9d29-acde48001122"].(*pb.VASP)
+
+	// Start the gRPC client
+	require.NoError(s.grpc.Connect())
+	defer s.grpc.Close()
+	client := api.NewTRISADirectoryClient(s.grpc.Conn)
+
+	// Emails need to be filled in for a valid VASP registration. Need to make copies
+	// of the contacts here to avoid modifying the fixtures for other tests.
```

Comment:
It might be easier to modify one of the fixtures so that all the contacts contain an email.